tests/Operation/WatchFunctionalTest.php

```diff
@@ -1596,9 +1567,9 @@ private function insertDocument($document): void
         $this->assertEquals(1, $writeResult->getInsertedCount());
     }
 
-    private function isPostBatchResumeTokenSupported()
+    private function skipIfPostBatchResumeTokenNotSupported(): void
     {
-        return version_compare($this->getServerVersion(), '4.0.7', '>=');
+        $this->skipIfServerVersionNotSupported('>=', '4.0.7', 'postBatchResumeToken is supported');
```

Comment:
As noted above, this method doesn't cover all use cases for `isPostBatchResumeTokenSupported()`. Three tests are skipped when `postBatchResumeToken` _is_ supported, and two are skipped when it's _not_ supported.

If you'd like to refactor this, I'd propose just dropping this method entirely and replacing the calling code with the appropriate `$this->skipIfServerVersion()` call. It's unlikely that we'd add many more tests for the feature so I'm not concerned with duplicating "4.0.7" in a few places.

Reply:
Good idea, I've totally removed `isPostBatchResumeTokenSupported`. This has more sense.